hyperspy/io_plugins/image.py

```diff
@@ -38,15 +38,16 @@
 
 # TODO Extend it to support SI
 def file_writer(filename, signal, file_format='png', **kwds):
-    """Writes data to any format supported by PIL
+    """Writes data to any format supported by imageio (PIL/pillow).
 
-        Parameters
-        ----------
-        filename: str
-        signal: a Signal instance
-        file_format : str
-            The fileformat defined by its extension that is any one supported by
-            PIL.
+    Parameters
+    ----------
+    filename: str
+    signal: a Signal instance
+    file_format : str
+        The fileformat defined by its extension that is any one supported by
+        imageio (PIL/pillow), for a list see 
+        https://imageio.readthedocs.io/en/stable/formats.html.
```

Comment:
While you are at it, is it worth to add passing kwds to `imageio.imwrite` and update the docstring/user guide accordingly?

Reply:
I just stumbled across the broken links, but you are right it totally makes sense. I will pass on `**kwds` both for the reader and writer.

Any idea, why the `file_format` parameter is even there if it is not passed on? Maybe some legacy issue? Normally, the format will be determined automatically, but if wanted the `format` argument can be passed as keyword argument.

Reply:
Indeed, this is legacy and it is fine to remove it.

Reply:
I already adapted that in the updated version.